packages/app/src/app/pages/Pro/legacy-pages/WorkspacePlanSelection.tsx

```diff
@@ -1,591 +1,284 @@
-                  // if it's already the same, then nothing to do here
-                  selectedPlan.billingInterval ===
-                    currentSubscription.billingInterval
-                }
-                onClick={() => setStep(Step.ConfirmBillingInterval)}
-                css={css({
-                  fontSize: 3,
-                  height: 10,
-                  fontFamily: 'Lato, sans-serif',
-                  fontWeight: 700,
-                })}
-              >
-                Update billing interval
-              </Button>
-              {currentSubscription.billingInterval ===
-                SubscriptionInterval.Yearly &&
-              selectedPlan.billingInterval === SubscriptionInterval.Monthly ? (
-                <Text align="center">
-                  Changing billing interval from Yearly to Monthly is not
-                  supported yet. Please email us at support@codesandbox.io
+          {isPersonalSpace ? (
+            <SubscriptionCard
+              title={
+                user.subscription.plan === 'patron' ? 'Patron' : 'Personal Pro'
+              }
+              features={PERSONAL_FEATURES}
+              cta={personalProCta}
+              isHighlighted
+            >
+              <Stack gap={1} direction="vertical">
+                <Text size={32} weight="500">
+                  ${user.subscription.amount}
                 </Text>
-              ) : null}
-            </>
+                {user.subscription.duration === 'yearly' ? (
+                  <Text>
+                    charged annually on{' '}
+                    {format(new Date(user.subscription.since), 'MMM dd')}
+                  </Text>
+                ) : (
+                  <Text>
+                    charged on the{' '}
+                    {format(new Date(user.subscription.since), 'do')} of each
+                    month
+                  </Text>
+                )}
+              </Stack>
+            </SubscriptionCard>
           ) : (
-            <Button
-              loading={loading}
-              disabled={
-                // non-admins can't upgrade
-                activeUserAuthorization !== TeamMemberAuthorization.Admin
-              }
-              onClick={() => setStep(Step.InlineCheckout)}
-              css={css({
-                fontSize: 3,
-                height: 10,
-                fontFamily: 'Lato, sans-serif',
-                fontWeight: 700,
-              })}
+            <SubscriptionCard
+              title="Team Pro"
+              features={TEAM_PRO_FEATURES}
+              cta={teamProCta}
+              isHighlighted
             >
-              Continue
-            </Button>
+              <Stack gap={1} direction="vertical">
+                <Text size={32} weight="500">
+                  ${subscription.unitPrice}
+                </Text>
+                <Text>
+                  <div>per editor</div>
+                  {isTeamAdmin ? (
```

Comment:
Humm if the given user is not an admin, will they see an empty field? I'd like to suggest abstracting differently. Instead of is personal or not, what if:

```jsx
{isPersonalSpace && </>}
{!isPersonalSpace && isTeamAdmin && </>}
```

Reply:
One less nested conditional, btw

Reply:
I think in the new pro pages we don't share as much information with non-admins so I chose to not show pricing details for regular team members.